Jenkinsfile

```diff
@@ -0,0 +1,135 @@
+@Library('pipeline-library')
+import com.genesys.jenkins.Service
+
+def notifications = null
+
+def isReleaseBranch() {
+    return env.SHORT_BRANCH.equals('main');
+}
+
+pipeline {
+  agent { label 'dev_mesos_v2' }
+  options {
+    quietPeriod(480)
+    disableConcurrentBuilds()
+  }
+
+  environment {
+    NPM_UTIL_PATH = "npm-utils"
+    REPO_DIR = "repo"
+    SHORT_BRANCH = env.GIT_BRANCH.replaceFirst(/^origin\//, '');
+  }
+
+  tools {
+    nodejs 'NodeJS 12.13.0'
```

Comment:
i got an error on this one indicating nodejs is not a tool
is it working 4 you?